doc/sphinx-guides/source/container/app-image.rst

```diff
@@ -0,0 +1,220 @@
+Dataverse Application Image
+===========================
+
+The application image is a layer on top of the base image and contains the Dataverse software.
+
+.. contents:: |toctitle|
+    :local:
+
+An "application image" offers you a deployment ready Dataverse application running on the underlying
+application server, which is provided by the :doc:`base-image`. Its sole purpose is to bundle the application
+and any additional material necessary to successfully jumpstart the application.
+
+Until all :ref:`jvm-options` are *MicroProfile Config* enabled, it also adds the necessary scripting glue to
+configure the applications domain during booting the application server. See :ref:`app-tunables`.
+
+Within the main repository, you may find the application image's files at ``<git root>/src/main/docker``.
+This is the same Maven module providing a Dataverse WAR file for classic installations, and uses the
+`Maven Docker Plugin <https://dmp.fabric8.io>`_ to build and ship the image within a special Maven profile.
+
+**NOTE: This image is created, maintained and supported by the Dataverse community on a best-effort basis.**
+IQSS will not offer you support how to deploy or run it, please reach out to the community for help on using it.
+You might be interested in taking a look at :doc:`../developers/containers`, linking you to some (community-based)
+efforts.
+
+
+
+Supported Image Tags
+++++++++++++++++++++
+
+This image is sourced from the main upstream code `repository of the Dataverse software <https://github.com/IQSS/dataverse>`_.
+Development and maintenance of the `image's code <https://github.com/IQSS/dataverse/tree/develop>`_ happens there
+(again, by the community).
+
+.. note::
+    Please note that this image is not (yet) available from Docker Hub. You need to build local to use
```

Comment:
Why not?

Reply:
Next sentence. :wink: See #9444 / #9447 This will be the next step and then this note will be replaced with something similar to what can be found in the base image docs.